Python/initconfig.c

```diff
@@ -1678,6 +1687,29 @@ config_read_env_vars(PyConfig *config)
     return _PyStatus_OK();
 }
 
+static PyStatus
+config_init_cpu_count(PyConfig *config)
+{
+    int cpu_count = -1;
```

Comment:
You can move the variable declaration inside the "if (sep)" block. Does it *have* to be initialized?

If you are afraid of undefined behavior, maybe config_wstr_to_int() should set ``*result`` to 0 on error.

Reply:
Now I added `PYTHONCPUCOUNT` envvar, so the current structure will be needed.

Reply:
If you want, you *can* move the variable declaration inside each "if (env)" block and duplicate the variable, to better show its scope. But well, that's just a personal preference. Feel free to ignore my coding style remark ;-)